appserver/pom.xml

```
@@ -150,7 +150,7 @@
                 <configuration>
                     <!--suppress UnresolvedMavenProperty -->
                     <dir>${stage.dir}/${install.dir.name}/glassfish/modules</dir>
-                    <excludes>jakarta.inject-api.jar,jakarta.servlet.jsp.jstl.jar,wasp.jar,jakarta.enterprise.concurrent.jar, jakarta.persistence.jar</excludes>
+                    <excludes>jakarta.inject-api.jar,jakarta.servlet.jsp.jstl.jar,wasp.jar,jakarta.persistence.jar</excludes>
```

I meant to replace it with the new artefact, not to remove it.
```suggestion
                    <excludes>jakarta.inject-api.jar,jakarta.servlet.jsp.jstl.jar,wasp.jar,concurro.jar,jakarta.persistence.jar</excludes>
```

---

Unless you have a reason for removing it?